### ISSUE TITLE
Alpine image for envoy

### ISSUE THREAD

**Issue by vijayendrabvs** [lyft/envoy#658]
Currently, the envoy container image is based off Ubuntu and is quite large (~500MB) and slows down quick deployment. Providing an option to build a minimal alpine image for envoy will be beneficial. I have a patch ready, will submit the PR and update this ticket with a link to it. Would be great if the community can take a look. Thanks!

**Comment by vijayendrabvs** [comment]
PR submitted at https://github.com/lyft/envoy/pull/659 . Will tag lyft team when tests pass.

**Comment by junr03** [comment]
@vijayendrabvs both alpine images are getting built, tested, and pushed in ci.

**Comment by vijayendrabvs** [comment]
Thanks @junr03 ! However, when I checked the lyft dockerhub repo, I realized I had missed out adding these two lines in the patch -

```
docker push lyft/envoy-alpine-debug:latest
docker push lyft/envoy-alpine:latest
```
in ci/docker_push.sh, so docker pull lyft/envoy-alpine will fail thus -

```
LM-SJL-11004971:~ vbhamidipati$ docker pull lyft/envoy-alpine
Using default tag: latest
Error response from daemon: manifest for lyft/envoy-alpine:latest not found
LM-SJL-11004971:~ vbhamidipati$
```

Can I submit a quick patch with the above two lines in a separate PR? Sorry for the inconvenience.

Also, can a note be added in the envoy tutorials to make a mention of the alpine image after the above correction patch goes in? Do I submit another GitHub request for that and/or where can I find the docs repo?

Thanks again!
